src/app/components/announcement-details/announcement-details.component.html

```diff
@@ -0,0 +1,21 @@
+<link rel="stylesheet" href="announcement-details.component.css">
```

Comment:
Nu-i nevoie sa adaugi link rel, stie componenta singura de unde sa isi ia styling-u. Se vede in fisieru .ts la styleUrls de mai jos
```suggestion
<link rel="stylesheet" href="announcement-details.component.css">

@Component({
    selector: 'app-announcement-details',
    templateUrl: './announcement-details.component.html',
    styleUrls: ['./announcement-details.component.css']
})
```